.github/workflows/ci.yml

```
@@ -0,0 +1,50 @@
+
+      - name: Lint
+        run: npx eslint .
+
+      - name: Unit tests
+        run: npm test -- --coverage
+
+      - name: Start dev server
+        run: npm run dev &
+        env:
+          CI: true
+        # Give server time to boot
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)